docker/properties-builder.sh

```diff
@@ -66,7 +66,9 @@ gitlab.projectIds=${GITLAB_PROJECT_IDS}
 # Another option: If using same username/password Jenkins auth,
 # set username/apiKey to use HTTP Basic Auth (blank=no auth)
 # A comma seperated list of api token corresponding to the project id mentioned above
-gitlab.apiKeys=${GITLAB_API_TOKENS}
+gitlab.apiKeys=${GITLAB_API_TOKEN}
```

Comment:
this has to be gitlab.apiKeys=${GITLAB_API_TOKENS} 
As we have multiple api tokens for build and deploy and only single one for scm collector